static/js/components/email/lib_test.js

```diff
@@ -273,16 +273,153 @@ describe('Specific email config', () => {
         });
       });
 
-      describe('findTerms', () => {
-        it('should return filters', () => {
-          let filters = findTerms(JSON.parse(queryFilters));
-          assert.equal(filters.length, 1);
-          assert.deepEqual(filters, [{
-            "program.enrollments.payment_status": "Paid"
-          }]);
+      describe('findFilters', () => {
+        it('should return an empty list if passed an empty object', () => {
+          let filters = findFilters({});
+          assert.equal(filters.length, 0);
```

Comment:
can also do `assert.deepEqual(filters, []);` which is slightly more readable IMO